<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Akash" ignored="0" name="Login with incorrect username and password" tests="3" failures="1" timestamp="13 Dec 2018 10:25:09 GMT" time="51.302" errors="0">
  <testcase name="TC001_test" time="26.028" classname="testScripts.MyAccount.Login.TC_002">
    <skipped/>
  </testcase> <!-- TC001_test -->
  <testcase name="TC001_test" time="11.271" classname="testScripts.MyAccount.Login.TC_002">
    <skipped/>
  </testcase> <!-- TC001_test -->
  <testcase name="TC001_test" time="9.621" classname="testScripts.MyAccount.Login.TC_002">
    <failure type="java.lang.AssertionError" message="expected [ERROR: Invalid username. Lost your password?] but found [ERROR: The password you entered for the username akash.srivastava is incorrect. Lost your password?]">
      <![CDATA[java.lang.AssertionError: expected [ERROR: Invalid username. Lost your password?] but found [ERROR: The password you entered for the username akash.srivastava is incorrect. Lost your password?]
at helper.assertion.AssertionHelper.verifyText(AssertionHelper.java:20)
at pageObjects.Login_RegisterPage.invalidUsernamePassword(Login_RegisterPage.java:86)
at testScripts.MyAccount.Login.TC_002.TC001_test(TC_002.java:21)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC001_test -->
</testsuite> <!-- Login with incorrect username and password -->
